locksmith/__tests__/utils/keyPricer.test.ts

```diff
@@ -26,8 +41,29 @@ jest.mock('@unlock-protocol/unlock-js', () => ({
 }))
 
 const keyPricer = new KeyPricer('provider url', 'unlock contract address')
+const spy = jest.spyOn(PriceConversion.prototype, 'convertToUSD')
 
 describe('KeyPricer', () => {
+  describe('keyPriceUSD', () => {
+    describe('when the lock currency has an exchange rate on coinbase', () => {
+      it('returns the key price in USD', async () => {
+        expect.assertions(1)
+        await keyPricer.keyPriceUSD('an address')
```

Comment:
Should we check what the return value is from these?

Reply:
@cnasc good question, the code generating the values has coverage under `PriceConversion` im proceeding as if this is an integration test. As long as this method calls the method with the correct params we'll assume its sorted of see a failure in a set of tests at the lower level